src/components/banner/index.tsx

```diff
@@ -82,71 +82,100 @@ export const Banner = (props: IBannerProps) => {
     cursor: 'pointer'
   };
   const defaultTextParentStyles = {
-    flex: '1'
+    flex: '1',
+    maxWidth: 'calc(100% - 80px)'
   };
 
   return (
-    <div
-      className="banner"
-      style={{
-        ...defaultBannerStyles,
-        ...BannerStyle
-      }}
-      onClick={onClickView}
-    >
+    <>
+      <style>
+        {`
+        @media screen and (max-width: 800px) {
+            .titleText {
+              overflow: hidden;
+              text-overflow: ellipsis;
+              max-height: 2.6em;
+              line-height: 1.3em;
+            }
+            .banner {
+              height: 250px;
+              display: flex;
+              flex-direction: column;
+            }
+          }
+        `}
+      </style>
       <div
+        className="banner"
         style={{
-          display: 'flex',
-          flexDirection: 'row'
+          ...defaultBannerStyles,
+          ...BannerStyle
         }}
+        onClick={onClickView}
       >
-        <div style={defaultTextParentStyles}>
-          <div style={{ overflowWrap: 'anywhere' }}>
-            <text style={{ ...defaultTitleStyles, ...titleStyle }}>
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'row'
+          }}
+        >
+          <div style={defaultTextParentStyles}>
+            <text
+              className="titleText"
+              style={{
+                ...defaultTitleStyles,
+                ...titleStyle,
+                display: 'block'
```

Comment:
display: block should go into `defaultTitleStyles`

Reply:
Done